src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -638,9 +638,9 @@ export class StudyViewPageStore {
 
     private geneMapCache:{[entrezGeneId:number]:string} = {};
 
-    @observable private chartsDimension: { [uniqueKey: string]: ChartDimension } = {};
+    @observable public chartsDimension = observable.map<ChartDimension>();
```

Comment:
i know this was existing code but I think we should change this to chartDimensions (plural at end).  It hard to see s in the middle.  Is an array of ChartDimension so ...